adapters/roulax/roulax.go

```diff
@@ -0,0 +1,138 @@
+package roulax
+
+import (
+	"encoding/json"
+	"net/http"
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+	"strings"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+
+
+
```

Comment:
Please remove unnecessary line space. Just one liner should be fine to distinguish between two code blocks.